### PR TITLE
tasks: Move to Fedora 37, drop container script labels

### PR DESCRIPTION
**Pull request by martinpitt**
I built this locally and verified that I can build and test cockpit with chromium and firefox.

  - [x] [build](https://github.com/cockpit-project/cockpituous/actions/runs/3326429810) and deploy
  - [x] test with anaconda: https://github.com/rhinstaller/anaconda/pull/4399
  - [x] test with c-machines: https://github.com/cockpit-project/cockpit-machines/pull/838
  - [x] https://github.com/cockpit-project/cockpit/pull/17847

### CODE REVIEW

**Comment by martinpitt** [comment]
What the... why does the [rebuild](https://github.com/cockpit-project/cockpituous/actions/runs/3326351200/jobs/5500036953) try to install stuff from rawhide!? My local build is fine, it uses f37 as the `FROM` line tells it to. Same as in [tests/tasks](https://github.com/cockpit-project/cockpituous/actions/runs/3326314113/jobs/5499911376#step:5:18) here, that was also as expected.

File: .github/workflows/build-images.yml [hunk 1]
```diff
@@ -7,15 +7,12 @@ jobs:
     runs-on: ubuntu-22.04
     environment: quay.io
     timeout-minutes: 30
-    # needs to use docker, as podman-built images are not accepted to RHEL 8's docker any more
```
Comment (martinpitt) [inline .github/workflows/build-images.yml:10]: FTR, we don't need them in RHEL 8 any more, our infra now runs on Fedora CoreOS.

**Comment by martinpitt** [comment]
There's one regression, cockpit's `TestRunTest.testRetry` now [fails](https://cockpit-logs.us-east-1.linodeobjects.com/pull-17839-20221026-054748-c5ecd1ca-arch/log.html#307-2). But this is subtle and easy enough to fix. I sent a PR, blocking on that.

**Review by marusak — APPROVED** [review]
Thank you!

**Pull request merged**